src/artm/core/protobuf_helpers.h

```diff
@@ -49,6 +50,38 @@ void repeated_field_append(T* field, int index, V value) {
   field->Set(index, new_value);
 }
 
+template<class T>
+std::vector<bool> is_member(T a, T b) {
```

Comment:
- Pass arguments by reference to avoid unnecessary  copying 
- Rename argument names: "a" to "elements" and "b" to "set" (I think matlab's names "a" and "b" are terribly bad - I always need to go to documentation to figure out what order should I use). You may also add a comment that "is_member finds which elements are present in the set. The resulting vector has the same size as elements size".

template<class T>
std::vector<bool> is_member(const T&elements, const T&set)